tests/openfl/component/aggregator/test_aggregator.py

```diff
@@ -171,80 +171,3 @@ def test_collaborator_task_completed_true(agg):
         col1, task_name, round_num)
 
     assert is_completed is True
-
-
-def test_is_task_done_no_cols(agg):
-    """Test that is_task_done returns True without corresponded collaborators."""
-    task_name = 'test_task_name'
-    agg.assigner.get_collaborators_for_task = mock.Mock(return_value=[])
-    is_task_done = agg._is_task_done(task_name)
-
-    assert is_task_done is True
-
-
-def test_is_task_done_not_done(agg):
-    """Test that is_task_done returns False in the corresponded case."""
-    task_name = 'test_task_name'
-    col1 = 'one'
-    col2 = 'two'
-    agg.assigner.get_collaborators_for_task = mock.Mock(return_value=[col1, col2])
-    is_task_done = agg._is_task_done(task_name)
-
-    assert is_task_done is False
-
-
-def test_is_task_done_done(agg):
-    """Test that is_task_done returns True in the corresponded case."""
-    round_num = 0
-    task_name = 'test_task_name'
-    col1 = 'one'
-    col2 = 'two'
-    agg.assigner.get_collaborators_for_task = mock.Mock(return_value=[col1, col2])
-    agg.collaborator_tasks_results = {
-        TaskResultKey(task_name, col1, round_num): 1,
-        TaskResultKey(task_name, col2, round_num): 1
-    }
-    is_task_done = agg._is_task_done(task_name)
-
-    assert is_task_done is True
-
-
-def test_is_round_done_no_tasks(agg):
-    """Test that is_round_done returns True in the corresponded case."""
-    agg.assigner.get_all_tasks_for_round = mock.Mock(return_value=[])
-    is_round_done = agg._is_round_done()
-
-    assert is_round_done is True
-
-
-def test_is_round_done_not_done(agg):
-    """Test that is_round_done returns False in the corresponded case."""
-    round_num = 0
-    task_name = 'test_task_name'
-    col1 = 'one'
-    col2 = 'two'
-    agg.assigner.get_all_tasks_for_round = mock.Mock(return_value=[task_name])
-    agg.assigner.get_collaborators_for_task = mock.Mock(return_value=[col1, col2])
-    agg.collaborator_tasks_results = {
-        TaskResultKey(task_name, col1, round_num): 1,
-    }
-    is_round_done = agg._is_round_done()
-
-    assert is_round_done is False
-
-
-def test_is_round_done_done(agg):
-    """Test that is_round_done returns True in the corresponded case."""
-    round_num = 0
-    task_name = 'test_task_name'
-    col1 = 'one'
-    col2 = 'two'
-    agg.assigner.get_all_tasks_for_round = mock.Mock(return_value=[task_name])
-    agg.assigner.get_collaborators_for_task = mock.Mock(return_value=[col1, col2])
-    agg.collaborator_tasks_results = {
-        TaskResultKey(task_name, col1, round_num): 1,
-        TaskResultKey(task_name, col2, round_num): 1
-    }
-    is_round_done = agg._is_round_done()
-
-    assert is_round_done is True
```

Comment:
Shouldn't these tests be compatible (i.e. passing) with the "happy" scenario of the `CutoffTimeBasedStragglerHandling` policy?

Reply:
Removed the unused `_is_round_done()` and `_is_task_done()` methods from the Aggregator, as they were identified as dead code. Additionally, updated the test cases to remove any references to these removed methods.